examples/web_agent/worker.py

```diff
@@ -0,0 +1,203 @@
+
+        # Prompt the user to log in if not already logged in
+        if not is_user_logged_in:
+            print(
+                f"Log into your Glassdoor account. Press Enter once done. Live view URL: {window_info.data.live_view_url}"
+            )
+            input()
+            print(
+                f"To avoid logging in again, use the profileId next time: {session.data.profile_id}"
+            )
+        else:
+            print(
+                f"User is already logged in. Live view URL: {window_info.data.live_view_url}"
+            )
+
+        # Navigate to the target URL
+        print("Navigating to target URL...")
+        client.windows.load_url(session_id, window.data.window_id, url=TARGET_URL)
+        print("Prompting the AI agent, waiting for a response...")
+
+        extract_data_response = client.windows.page_query(
+            session_id,
+            window.data.window_id,
+            prompt=EXTRACT_DATA_PROMPT,
+            configuration=PageQueryConfig(output_schema=EXTRACT_DATA_OUTPUT_SCHEMA),
+        )
+        formatted_json = json.dumps(
+            json.loads(extract_data_response.data.model_response), indent=2
+        )
+
+        # check if db exists, if not then it'll be created
+        try:
+            print("Existing workers")
+            print(dataframe_from_query_dev("SELECT * from workers limit 5"))
+        except:
+            for statement in sql_statements:
+                insert_tables(statement)
+
+        workers = find_workers(TARGET_URL, EXTRACT_DATA_PROMPT)
+        if not workers:
+            create_worker([(TARGET_URL, EXTRACT_DATA_PROMPT)])
+        print("Response:\n\n", formatted_json)
+
+        old_content = retrieve_previous_result(TARGET_URL, EXTRACT_DATA_PROMPT)
+        if old_content:
+            promptContentResponse = client.windows.page_query(
+                session.data.id,
+                window_info.data.window_id,
+                prompt=comparison_prompt(old_content, formatted_json),
+            )
+            print("Comparison result: ")
+            print(promptContentResponse.data.model_response)
+            insert_comparison(
+                TARGET_URL,
+                EXTRACT_DATA_PROMPT,
+                promptContentResponse.data.model_response,
+            )
+
+        insert_result(TARGET_URL, EXTRACT_DATA_PROMPT, formatted_json)
+
+    except Exception as e:
+        try:
+            print(e.status_code, e.message, e.body)
+        except:
+            print(e)
+
+    finally:
+        # Clean up
+        if client is not None and session_id is not None:
+            if (
+                window is not None
+                and window.data is not None
+                and window.data.window_id is not None
+            ):
+                client.windows.close(session_id, window.data.window_id)
+            client.sessions.terminate(session_id)
+            print("Session terminated")
+
+
+if __name__ == "__main__":
+    asyncio.run(run())
```

Comment:
So you can remove the asyncio import, change to:
```suggestion
    run()
```

Reply:
Done